Sources/Signal.swift

```diff
@@ -844,6 +850,9 @@ extension SignalProtocol {
 				switch receivedEvent {
 				case let .value(v):
 					value?(v)
+					observer.send(value: v)
```

Comment:
I'm confused, why is this line added here too? Aren't we sending events twice now?